docs/components/console/manage-clusters/cluster-capacity.md

```diff
@@ -0,0 +1,102 @@
+---
+id: cluster-capacity
+title: Cluster capacity
+description: "Cluster capacity provides you with a high-level overview of how well a cluster is coping with and handling its current workload."
+---
+
+import CapacityImg from './img/cluster-capacity.png';
+import CapacityPercentImg from './img/cluster-capacity-percent.png';
+
+View and manage your cluster capacity and health.
+
+## About cluster capacity
+
+Cluster capacity provides a high-level overview of how well a cluster is coping with its current workload.
+
+- Use this information to check and monitor if a cluster is appropriately sized for its workload.
+- Cluster capacity can also be used as an indicator of cluster health. For example, a cluster running at maximum capacity can be an indicator of poor cluster responsiveness.
+
+A general guideline to follow when using cluster capacity as a metric is:
+
+- **High capacity**: The higher the capacity, the more likely it is that things will slow down, time out, requests will fail, and so on.
+- **Low capacity**: The lower the capacity, the more you could increase the cluster workload as the cluster is probably underused.
```

Comment:
This is when using word "capacity" becomes tricky. 
IMHO, high capacity means the load on the clusters is low, i.e. low %. Low capacity - the opposite, i.e. high load. 
I've noticed it also in other parts of this page.

Reply:
Agreed, but as we had left capacity on the page, I thought it was less confusing to keep the same wording here than to talk about load, when load never appears anywhere else.

I'm all for changing it though, but we need to change it on the frontend too.